lesson3/README.md

```diff
@@ -1,53 +1,47 @@
 # Lesson 3: Data Models, Relationships, and Schemas
 
-In the final week, additional theory will be covered to discuss more complex relational data. Students will learn about entity relationship modelling and how to convert these models to a database schema using normalisation and foreign-key constraints. Non-relational data will also be considered, as well as the benefits and drawbacks of relational and non-relational models.
+In the final week, the students will be introduced to SQL injection and how to use MySQL permissions to avoid it. MongoDB, as an example of a non-relational database, will also be introduced, as well as the benefits and drawbacks of relational and non-relational models.
```

Comment:
This one is a little tough. We should definitely explain and demonstrate SQL injections, but the part about how to really avoid it is super difficult to explain when they haven't had nodejs yet :thinking: 

The wording around user permissions should probably be reworked, since it doesn't really prevent injections but only limits the potential damage. Still very important to explain of course.

How about something like this?

```suggestion
In the final week, the students will be introduced to SQL injections and common security practices around MySQL permissions. MongoDB, as an example of a non-relational database, will also be introduced, as well as the benefits and drawbacks of relational and non-relational models.
```

Reply:
The way you wrote it is much better, thank you! :D